addon/components/imgix-image-wrapped.js

```diff
@@ -1,10 +1,10 @@
 import Component from '@ember/component';
 import { deprecate } from '@ember/application/deprecations';
-import ResizeMixin from 'ember-resize-mixin/main';
+import ResizeAware from 'ember-resize-aware/mixins/resize-aware';
 import layout from '../templates/components/imgix-image-wrapped';
 import ImgixPathBehavior from '../mixins/imgix-path-behavior';
 
-export default Component.extend(ResizeMixin, ImgixPathBehavior, {
+export default Component.extend(ResizeAware, ImgixPathBehavior, {
```

Comment:
Does this preserve `imgix-image-wrapped`s behaviour?

Reply:
Yes, 
That lib is a fork of ember-resize-mixin but updated more recently. ( I maintain it so I can add you as collab if you want ) Ember-resize-mixin is not maintained.

Reply:
Great stuff. It would be good to be added as a collaborator 👍 Thanks @Duder-onomy